regml.py

```diff
@@ -11,6 +11,7 @@
 
 import click
 from lxml import etree
+from lxml import objectify
```

Comment:
More I think about it, the more I think we should stick with etree for now... Don't want to introduce potential API weirdness (even though I don't think there be any).

You should be able to accomplish the same thing with:

```
parser = etree.XMLParser(huge_tree=True)
tree = etree.fromstring(xml_str, parser)
```